### PR TITLE
README: Cleanup

### PR DESCRIPTION
**Pull request by raziel-**

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -153,22 +159,30 @@ Compression Tools:
...
+                NOTE: Specify --mac for the mac version (obviously).
                 Default output is input with changed extension.
 
         compress_gob
                 Compresses Gobliiins! data files.
 
+                Example of usage:
+                ./scummvm-tools-cli --tool compress_gob [-o outputpath] -f <conf_file>
+                
+                <conf file> is a .gob file generated extract_gob_stk.
```
Comment (rootfather) [inline README]: are you missing a "by" here?

File: README [hunk 2]
```diff
@@ -229,20 +244,20 @@ Compression Tools:
                 Example of usage:
                 ./scummvm-tools-cli --tool compress_scumm_bun [mode params] [-o outputfile] <infile>
 
-                Default outputfile is inputfile with the extension ".bun"
+                Default outputfile is inputfile with the extension ".bun".
 
-                Please note that FLAC compression will produce larger files
-                than the original, for The Curse of Monkey Island!
+                Note: FLAC compression will produce larger files than the original,
```
Comment (rootfather) [inline README]: I think a better style would be
`Note: For "The Curse of Monkey Island", FLAC compression will produce larger files than in the original game.`

File: README (hunk 1 above)
Comment (raziel-) [inline README]: Yep, c&p directly from the command line help

File: README [hunk 3]
```diff
@@ -267,58 +282,64 @@ Compression Tools:
...
                 Once compressed, only TOUCHE.DAT and TOUCHE.SOx files are
                 required to play the game under ScummVM.
 
                 Example of usage:
                 ./scummvm-tools-cli --tool compress_touche [mode params] [-o outputfile] <inputdir>
 
-                Default outpufile is TOUCHE.* (depends on compression method).
-                Files in the input folder should be in uppercase.
+                NOTE: Default outpufile is TOUCHE.* (depending on the compression method).
+                Files in the input folder should be uppercase.
```
Comment (rootfather) [inline README]: should be "outputfile"

**Comment by rootfather** [comment]
Can you please squash all this commits to a single one? Once that's done, I'll merge this PR.

**Comment by raziel-** [comment]
Not sure how to do it on the website.

I could close and open a new PR with all the changes in one, if that is what you need?

I've only got rudimentary git support and do everything on the github website

**Comment by rootfather** [comment]
sev told me that there is an option to squash the commits before merging, so I did that for you. Thanks for your work!

**Pull request merged**